scripts/spidhub-fargate/environments/dev/storage/config/spid-saml-check/v1/configvolume/spid_users.json

```diff
@@ -0,0 +1,562 @@
+		"ivaCode": "12345678987",
+		"companyName": "Scuola magistrale Montessori",
+		"registeredOffice": "Via Listz 21 00144 Roma",
+		"companyFiscalNumber": "TINIT-12345678987",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "little",
+		"password": "password123",
+		"purposeIDType": "1",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-015",
+		"familyName": "Little",
+		"name": "Stuart",
+		"gender": "M",
+		"dateOfBirth": "2016-12-12",
+		"countyOfBirth": "RM",
+		"placeOfBirth": "H501",
+		"idCard": "cartaIdentita AC137100 comuneRoma 2020-05-22 2023-05-22",
+		"expirationDate": "2022-05-22",
+		"fiscalNumber": "TINIT-LTTSRT16T12H501Y",
+		"mobilePhone": "300000000",
+		"email": "topolinoingamba@gmail.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "dino",
+		"password": "password123",
+		"spidLevel": 1,
+		"disabled": true,
+
+		"spidCode": "SPID-016",
+		"familyName": "De Sauro",
+		"name": "Dino",
+		"gender": "M",
+		"dateOfBirth": "0000-00-00",
+		"countyOfBirth": "BA",
+		"placeOfBirth": "A225",
+		"idCard": "cartaIdentita JUR4SS1C comuneAltamura 0000-00-00 0000-00-00",
+		"expirationDate": "0001-00-00",
+		"fiscalNumber": "TINIT-DSRDNI00A01A225I",
+		"mobilePhone": "000000000",
+		"email": "noreply@estinti.paleo",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "",
+		"domicileStreetAddress": "",
+		"domicilePostalCode": "",
+		"domicileMunicipality": "",
+		"domicileProvince": "",
+		"domicileNation": ""
+	},
+	{
+		"username": "galileo",
+		"password": "password123",
+		"purposeIDType": "3",
+		"spidLevel": 1,
+
+		"spidCode": "SPID-017",
+		"familyName": "Galilei",
+		"name": "Galileo",
+		"gender": "M",
+		"dateOfBirth": "1964-02-15",
+		"countyOfBirth": "PI",
+		"placeOfBirth": "G702",
+		"idCard": "cartaIdentita PI150264 comunePisa 2003-07-11 2023-07-11",
+		"expirationDate": "2064-02-15",
+		"fiscalNumber": "TINIT-GLLGLL64B15G702I",
+		"mobilePhone": "3479999999",
+		"email": "ilcannocchiale@libero.it",
+		"digitalAddress": "ilcannocchiale@pec.agid.gov.it",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via delle scienze 1 56121 Pisa",
+		"domicileStreetAddress": "Via delle scienze 1",
+		"domicilePostalCode": "56121",
+		"domicileMunicipality": "Pisa",
+		"domicileProvince": "PI",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "leonardo",
+		"password": "password123",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)